frontend/index.html

```diff
@@ -759,6 +828,8 @@ <h2 data-i18n="contact">Contact</h2>
             } else {
                 hideTokenAndURL();
             }
+            // only place I've been able to ensure the right data exists before proceeding
+            SUPPORTED_LANGUAGES = getSupportedLanguages();
```

Comment:
This is the best place I've found to load the supported languages, since the data from the config file will exist here

Reply:
The issue I've been having is that since the supported languages come from the dynamic config, we don't know which ones to load until loading the dynamic config file is loaded, so I think we need to wait for the dynamic config to be done loading before we load any languages -- this will allow for configuring the default too!

I am generally familiar with what promises are in Javascript, but haven't worked with `async` and `await` yet, I'm looking into those now and plan to try using `await` for dynamic config since all language info not depends on that.

Reply:
> The issue I've been having is that since the supported languages come from the dynamic config, we don't know which ones to load until loading the dynamic config file is loaded, so I think we need to wait for the dynamic config to be done loading before we load any languages -- this will allow for configuring the default too!

Correct. Now all promises need to be moved into the `then` for the dynamic load.

> I'm looking into those now and plan to try using await for dynamic config since all language info now depends on that.

Yup! Note that you can only use `await` in an `async` function or a top-level module. Not sure if `document.ready()` qualifies, but it seems like it should be the toppest of all top modules 😄

Reply:
https://stackoverflow.com/questions/69984178/async-await-with-jquery-document-ready

Reply:
I just reconfigured, but there's still a bug somewhere! I don't have time to find it right now, but will tonight or tomorrow! Everything loads and displays in the right languages at the right times (behaving as expected) with the exception of the `language_name` where it's defaulting to English - I find it odd that all the other keys load fine and the page is 99% translated, but I'll look into it when I get a moment

Reply:
I wrapped up the reconfiguring of the load sequences -- now it loads the dynamic config with `await`, then the default (first in config) i18n file with `await`, and finally loads the other language(s) with a traditional promise. In the `then` for the other language loads, the language menu is set up and enabled. I tried using `await` for all three load processes, but that was yielding the issue from yesterday - this way works consistently.

I also added the `dashboard-preview-lo.png` which I noticed was missing in debugging.